target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -513,6 +514,8 @@ ramips_setup_macs()
 	alfa-network,w502u|\
 	arcwireless,freestation5|\
 	cudy,wr1000|\
+	netgear,ex3700|\
```

Comment:
Maybe adding the WAN mac address to ex3700 should also be in a separate commit.

Reply:
Done

Reply:
Now you mixed up network and MAC address setup....